packages/manager/apps/web/client/app/app.js

```diff
@@ -40,6 +40,7 @@ import config from './config/config';
 import domainEmailObfuscation from './domain/email-obfuscation/index';
 import domainOptin from './domain/optin/index';
 import domainDnsZone from './dns-zone';
+import hostingWebsiteCoach from './hosting/website-coach/website-coach.module';
```

Comment:
Usually the name is `index.js` so you can
```suggestion
import hostingWebsiteCoach from './hosting/website-coach';
```

:)

Reply:
When I've seen that it's use `index.js` (for the lazy load), I've asked what the preferred form. And it's `website-coach.module.js` ;)